exercises/word-search/src/example/java/Pair.java

```diff
@@ -0,0 +1,39 @@
+class Pair {
+
+    private final int x;
+
+    private final int y;
+
+    Pair(final int x, final int y) {
+        this.y = y;
+        this.x = x;
+    }
+
+    int getX() {
+        return x;
+    }
+
+    int getY() {
+        return y;
+    }
+
+    @SuppressWarnings("SimplifiableIfStatement")
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Pair pair = (Pair) o;
+
+        if (x != pair.x) return false;
+        return y == pair.y;
```

Comment:
That's strange that it would generate it like that... So do you want to simplify it? :)